src/pudl/settings.py

```diff
@@ -15,6 +16,16 @@
 from pudl.metadata.resources.eia861 import TABLE_DEPENDENCIES
 
 
+class XbrlFormNumber(Enum):
+    """Contains full list of supported FERC XBRL forms."""
+
+    FORM1 = 1
+    FORM2 = 1
```

Comment:
This looks like a typo? I think `FORM2 = 2` probably.

You can add the `@enum.unique` decorator to enforce this kind of thing.

Reply:
Good to know. Fixed and added the decorator